### PR TITLE
[WFCORE-846] Some tests throws "Permission check failed" while run with security manager

### PR DESCRIPTION
**Pull request by iweiss**
Issue: https://issues.jboss.org/browse/WFCORE-846

### CODE REVIEW

**Comment by iweiss** [comment]
retest this please

File: testsuite/shared/pom.xml [hunk 1]
```diff
@@ -165,6 +165,21 @@
             <artifactId>byteman</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.io7m.xom</groupId>
+            <artifactId>xom</artifactId>
+            <version>1.2.10</version>
```
Comment (bstansberry) [inline testsuite/shared/pom.xml]: Use dependency-management in the root pom to specify versions, driven by a new version.xxxx property.

**Comment by bstansberry** [comment]
CI failures are related.

File: pom.xml [hunk 2]
```diff
@@ -143,6 +143,7 @@
         <!-- Non-default maven plugin versions and configuration -->
         <version.org.zanata.plugin>3.9.1</version.org.zanata.plugin>
         <version.xml.plugin>1.0.1</version.xml.plugin>
+        <version.com.io7m.xom>1.2.10</version.com.io7m.xom>
```
Comment (bstansberry) [inline pom.xml]: These are kept in alphabetical order.

**Comment by bstansberry** [comment]
I suspect these failures are due to the same sort of thing you already fixed in a couple test classes so the fix would be to apply the same thing.

**Comment by iweiss** [comment]
@bstansberry That's correct. The deployment test issues are fixed on another PR, IIRC.

**Comment by iweiss** [comment]
retest this please

**Pull request merged**